.github/workflows/bindings_python.yml

```diff
@@ -122,6 +122,8 @@ jobs:
       - uses: actions/download-artifact@v3
         with:
           name: wheels
+          path: bindings/python/dist
```

Comment:
I'm always confused with `actions/download-artifact`.

If we upload an artifact with name `abc` and path `bindings/python/dist`, what's the layout when we download it?

Reply:
It will be downloaded to a subdir with artifact name.

Reply:
> It will be downloaded to a subdir with artifact name.

Got it, so we will have all artifacts under `wheels/`?

Reply:
I think so.

Reply:
Thanks for the clarification!